### PR TITLE
Update header.ejs

### PR DESCRIPTION
**Pull request by WarpWing**
Small changes to header.ejs

### CODE REVIEW

**Review by DuckySoLucky — APPROVED** [review]
lgtm, didn't test

File: includes/header.ejs [hunk 1]
```diff
@@ -33,8 +33,8 @@
     <div id="info_box_content">
       <p>SkyCrypt is a free open-source stats viewer for Hypixel SkyBlock.</p>
       <p>You can report bugs, suggest features, or contribute to the code on <a class="link link-github" href="https://github.com/SkyCryptWebsite/SkyCrypt" target="_blank" rel="noreferrer">GitHub</a>. It would be much appreciated!</p>
-      <p>Join our community on <a class="link link-discord" href="https://discord.gg/cNgADv2kEQ">Discord</a></p>
-      <p>Help keep SkyCrypt ad free by donating on <a class="link link-patreon" href="https://www.patreon.com/shiiyu">Patreon</a></p>
+      <p>Join our community on <a class="link link-discord" href="https://discord.gg/cNgADv2kEQ"target="_blank" rel="noreferrer">Discord!</a></p>
```
Comment (soopyc) [inline includes/header.ejs]:
```suggestion
      <p>Join our community on <a class="link link-discord" href="https://discord.gg/cNgADv2kEQ" target="_blank" rel="noreferrer">Discord!</a></p>
```

File: includes/header.ejs [hunk 2]
```diff
@@ -33,8 +33,8 @@
     <div id="info_box_content">
       <p>SkyCrypt is a free open-source stats viewer for Hypixel SkyBlock.</p>
       <p>You can report bugs, suggest features, or contribute to the code on <a class="link link-github" href="https://github.com/SkyCryptWebsite/SkyCrypt" target="_blank" rel="noreferrer">GitHub</a>. It would be much appreciated!</p>
-      <p>Join our community on <a class="link link-discord" href="https://discord.gg/cNgADv2kEQ">Discord</a></p>
-      <p>Help keep SkyCrypt ad free by donating on <a class="link link-patreon" href="https://www.patreon.com/shiiyu">Patreon</a></p>
+      <p>Join our community on <a class="link link-discord" href="https://discord.gg/cNgADv2kEQ" target="_blank" rel="noreferrer">Discord!</a></p>
+      <p>Help keep SkyCrypt ad free by donating on <a class="link link-patreon" href="https://www.patreon.com/shiiyu">Patreon target="_blank" rel="noreferrer"!</a></p>
```
Comment (soopyc) [inline includes/header.ejs]:
```suggestion
      <p>Help keep SkyCrypt ad free by donating on <a class="link link-patreon" href="https://www.patreon.com/shiiyu" target="_blank" rel="noreferrer">Patreon!</a></p>
```

**Comment by soopyc** [comment]
lgtm

**Review by MartinNemi03 — APPROVED** [review]
lgtm, didn't test

**Pull request merged**